google/ads/google_ads/client.py

```diff
@@ -38,16 +38,24 @@
 _VALID_API_VERSIONS = ['v1']
 _DEFAULT_VERSION = _VALID_API_VERSIONS[0]
 _REQUEST_ID_KEY = 'request-id'
+_ENV_PREFIX = 'GOOGLE_ADS_'
+_KEYS_ENV_VARIABLES_MAP = {
+    key: _ENV_PREFIX + key.upper() for key in
+    list(_REQUIRED_KEYS) + ['login_customer_id', 'endpoint', 'logging']
```

Comment:
again, remove endpoint

Reply:
We probably want to keep endpoint here, otherwise if we're configuring the lib with env vars we can't set the endpoint. I think it's reasonable not to document, but there should be symmetry with the k/vs you can set in each mode of configuration, i.e. it doesn't make sense that endpoint can be set via yaml but not via an env var.

Reply:
Removed endpoint from env vars as it was originally requested. Should I restore it?

Reply:
Yes please, sorry about the confusion.

Reply:
Done